From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-vma-the-pgoff-is-correct-if-can_merge_right-fix
Date: Fri Oct 25 07:39:41 PM PDT 2024

remove now-unused local

mm/vma.c: In function 'vma_merge_new_range':
mm/vma.c:965:23: error: unused variable 'start' [-Werror=unused-variable]
  965 |         unsigned long start = vmg->start;
      |                       ^~~~~
Cc: Jann Horn <jannh@google.com>
Cc: Liam R. Howlett <Liam.Howlett@oracle.com>
Cc: Lorenzo Stoakes <lorenzo.stoakes@oracle.com>
Cc: Vlastimil Babka <vbabka@suse.cz>
Cc: Wei Yang <richard.weiyang@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vma.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/vma.c~mm-vma-the-pgoff-is-correct-if-can_merge_right-fix
+++ a/mm/vma.c
@@ -962,7 +962,6 @@ struct vm_area_struct *vma_merge_new_ran
 {
 	struct vm_area_struct *prev = vmg->prev;
 	struct vm_area_struct *next = vmg->next;
-	unsigned long start = vmg->start;
 	unsigned long end = vmg->end;
 	bool can_merge_left, can_merge_right;
 	bool just_expand = vmg->merge_flags & VMG_FLAG_JUST_EXPAND;
_
